smart_contracts/contract_as/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "casper-contract",
-  "version": "1.4.4",
+  "version": "1.4.5",
```

Comment:
Was this deliberate?

Reply:
I wanted to make sure it was bumped so that it doesn't block CI.